pkg/controller/kfservice/resources/knative/configuration.go

```diff
@@ -126,13 +129,15 @@ func (c *ConfigurationBuilder) CreateKnativeConfiguration(name string, metadata
 
 func configurationAnnotationFilter(annotationKey string) bool {
 	switch annotationKey {
```

Comment:
This switch statement may grow uncontrollably. Would it be cleaner to simply define a slice and then use slice.Contains(key)?